contracts/DolaFlashMinter.sol

```diff
@@ -9,29 +8,36 @@ import "./utils/Address.sol";
 import "./ERC20/IERC20.sol";
 import "./ERC20/SafeERC20.sol";
 
-contract DolaFlashMinter is Ownable, IERC3156FlashLender {
+abstract contract DolaFlashMinter is Ownable, IERC3156FlashLender {
```

Comment:
Why is this abstract now?

Reply:
This is to respect the intent of the previous code where we had hardcoded addresses, I wanted to make sure it was not possible to deploy without hardcoded addresses. I'll add a comment to make it more explicit.